README.md

```diff
@@ -1,3 +1,25 @@
+## Requirements
+
+* scrapy>=2.5.1
```

Comment:
For `scrapy`, we can add it to https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/setup.py#L13 without any version specifiers.

Reply:
Sure, I can modify that part as well. It makes sense. When we do install via pip it will download the latest version only unless we specify the version.

Reply:
I think one way to specify this in the doc could be how https://github.com/scrapy-plugins/scrapy-zyte-smartproxy does it.

Reply:
Yes I have modified the documentation for Requirement section as shown in the https://github.com/scrapy-plugins/scrapy-zyte-smartproxy